src/python/pants/backend/codegen/tasks/wire_gen.py

```diff
@@ -10,16 +10,14 @@
 import os
 from collections import OrderedDict, defaultdict
 
-from twitter.common.collections import OrderedSet, maybe_list
+from twitter.common.collections import OrderedSet
```

Comment:
I like the diff in this file.  Lots of red that is replaced with just a few simple overrides.